drafts/Stormless.md

```diff
@@ -0,0 +1,198 @@
+  
+  > PRO TIP: Use this hack to get your own token and use it for playing. Much faster. But never, never do it for production!  
+
+### Create a project, add a first action
+
+Try `sls --help` to make sure that at least something works. Now put your coffee aside, time to create a project. Some like using templates that come with `serverless create --template`. I prefer to start from scratch: 
+
+``` 
+mkdir slack-signup-serverless-stormless
+cd slack-signup-serverless-stormless
+
+npm init
+
+# Once you answer questions, the project is set up.
+
+```
+
+Now we are ready to do what we came for:[`serverless-plugin-stackstorm`](https://github.com/StackStorm/serverless-plugin-stackstorm). Install it:
+
+```
+npm install --save-dev serverless-plugin-stackstorm
+```
+
+Create the first action. I'll use a battle-tested [Slack pack from StackStorm Exchange](https://exchange.stackstorm.org/#slack). While StackStorm Exchange is not smart enough yet to show pack's action list, `sls stackstorm` will rescue us.
+
+`sls stackstorm info --pack slack` 
+
+Oh my! There're so many what are they? I guess I need a PR to print action description. Meantme, `| grep admin` will get us the one we need: `slack.users.admin.invite`. Let's inquire the action for it's parameters:
+
+```
+$ sls stackstorm info --action slack.users.admin.invite
+slack.users.admin.invite ...... Send an invitation to join a Slack Org
+Parameters
+  attempts [integer]  ......... description is missing
+  channels [string]  .......... Channels to auto join.
+  email [string] (required) ... Email address to send invitation to.
+  first_name [string]  ........ Users first name
+  set_active [boolean]  ....... Should the user be active.
+  token [string]  ............. Slack API token.
+Config
+  action_token [string]  ...... Slack Action token.
+  admin [object]  ............. Admin-action specific settings.
+  post_message_action [object]   Post message action specific settings.
+  sensor [object]  ............ Sensor specific settings.
+```
+
+
+Awesome! We can see that there is only one required parameter, `email`, but I'll add `first_name` to be conversational. The token can be passed as parameters, or as config. And if I choose to use config, my prior tribal knowledge hints that the `admin [object]` requires only `admin_token`. The very one I asked you to remember when you were setting up Slack workspace.
+> PRO TIP. While we are still polishing the plugin to expose all the Config details, you can find it out by exploring StackStorm Exchange pack config schema in `config.schema.yaml` file. For example, here is [config.example.yaml](https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/config.schema.yaml#L47-L78) for our Slack pack.
+
+Now we have all we need to create the heart of any Serverless project: the `serverless.yml`. Here it comes: 
+
+```
+service: signup-stormless
```

Comment:
Yeah, it's good this project name is relatively unique. Serverless throws a very unfriendly exception in case project with the same name already exists on that account (the error says S3 bucket doesn't exist, but the actual issue is conflicting project name).

Reply:
I considered adding a PRO tip but it's unlikely they'll step on this. Will leave it out.